tasks/assert.yml

```diff
@@ -0,0 +1,92 @@
+---
+- name: test if rsyslog_receiver is set correctly
+  ansible.builtin.assert:
+    that:
+      - mysql_replication_gtid is defined
+      - mysql_replication_gtid is boolean
+    quiet: yes
+  when:
+    - mysql_replication_gtid is defined
```

Comment:
This is not needed on every variables, only the non mandatory without default value or the complex ones. Take another look at Robert de Bock's work and you will see the distinction.

Here the variable `mysql_replication_gtid` has a default value and is this always defined.

Reply:
Thanks for the information, I just removed the when.